source/welcome/whats-new-in-v72.rst

```diff
@@ -0,0 +1,183 @@
+What’s new in Mattermost Boards v7.2
+=====================================
+
+Mattermost Boards v7.2 will be released in the week of August 15th, and all Cloud plans will be automatically upgraded to the latest version during this time.
+
+Boards is moving from a channel-based to a role-based permissions system. This means that access to individual boards can be controlled on a channel level or a board level, depending on whether a board is linked to a channel or not.
+
+TL;DR
+------
+
+1. All the boards you're currently a member of from your current team will appear on the sidebar without needing to switch workspaces.
+2. Organize boards on the sidebar with custom categories. 
+3. Press Ctrl+K / Cmd+K to find additional boards.
+4. Navigate between teams in Boards with the new team switcher.
+5. Set board and template permissions in the new **Share** dialog.
+6. Link boards to channels to automatically grant board permissions to channel members.
+
+Navigation
+----------
+
+Sidebar categories
+~~~~~~~~~~~~~~~~~~~
+
+Navigation is simplified, and now works similarly to channels. No more switching between workspaces - and the dashboard page has been removed. All the boards you're currently a member of from your current team will appear on the sidebar without needing to switch workspaces. Use the search box (Ctrl+K / Cmd+K) to find additional boards.
+
+.. image:: ../images/boards-whats-new-01.png
```

Comment:
@justinegeffen Can we reduce the width of this image to 303px so it's the same width as the 2nd image after this one?

Reply:
Sure. Let me know if that looks better.